src/main/java/uk/gov/hmcts/divorce/caseworker/schedule/CaseworkerAwaitingConditionalOrderTask.java

```diff
@@ -51,13 +51,21 @@ public void execute() {
                 try {
                     Map<String, Object> caseDataMap = caseDetails.getData();
                     CaseData caseData = objectMapper.convertValue(caseDataMap, CaseData.class);
-                    long weeksBetweenIssueDateAndNow = WEEKS.between(caseData.getIssueDate(), LocalDate.now());
-
-                    if (weeksBetweenIssueDateAndNow > HOLDING_PERIOD_IN_WEEKS) {
-                        log.info("Case id {} has been in holding state for > 20 weeks hence moving state to AwaitingConditionalOrder",
-                            caseDetails.getId()
+                    if (caseData.getIssueDate() == null) {
```

Comment:
could you please invert this and put this in else part
if(issue!=null){}
else{log...}

Reply:
Also it should log as error rather than info

Reply:
PMD rule failing with : 
Avoid negation within an "if" expression with an "else" clause. For example, rephrase: if (x != y) diff(); else same(); as: if (x == y) same(); else diff();.

Reply:
Changed logging to error.